apps/v1/consts.go

```diff
@@ -0,0 +1,135 @@
+package v1
+
+const (
+	// DeploymentStatusReasonAnnotation represents the reason for deployment being in a given state
+	// Used for specifying the reason for cancellation or failure of a deployment
+	// This is set by deployer controller and consumed in retry and history commands.
+	DeploymentStatusReasonAnnotation = "openshift.io/deployment.status-reason"
+
+	// DeploymentPodAnnotation is an annotation on a deployment (a ReplicationController). The
+	// annotation value is the name of the deployer Pod which will act upon the ReplicationController
+	// to implement the deployment behavior.
+	// This is set by deployer controller and consumed by oc adm top command.
+	DeploymentPodAnnotation = "openshift.io/deployer-pod.name"
+
+	// DeployerPodCreatedAtAnnotation is an annotation on a deployment that
+	// records the time in RFC3339 format of when the deployer pod for this particular
+	// deployment was created.
+	// This is set by deployer controller, but not consumed by any command or internally.
+	DeployerPodCreatedAtAnnotation = "openshift.io/deployer-pod.created-at"
+
+	// DeployerPodStartedAtAnnotation is an annotation on a deployment that
+	// records the time in RFC3339 format of when the deployer pod for this particular
+	// deployment was started.
+	// This is set by deployer controller, but not consumed by any command or internally.
+	DeployerPodStartedAtAnnotation = "openshift.io/deployer-pod.started-at"
```

Comment:
@tnozicka @deads2k candidate for removal

Reply:
why? or you just want to move it to appsutil?

Reply:
@tnozicka I think these are providing "future" value for us, not for users or anyone integrating with OpenShift... I think the intention here was to improve the deployer process by knowing the times here so we can calculate the timeouts better. That to me is an internal thing.

Reply:
@tnozicka yes, `appsutil`

Reply:
+1, appsutil only since it's internal